cmd/create/idp/cmd.go

```diff
@@ -434,6 +438,18 @@ func getIDPName(cmd *cobra.Command, idpName string, r *rosa.Runtime) string {
 	return strings.Trim(idpName, " \t")
 }
 
+func validateIDPName(idpName interface{}) error {
+
+	if !idRE.MatchString(idpName.(string)) {
```

Comment:
If a non string is passed into this function it will cause a panic. 

Can we ensure `idpName` is the expected type.

Reply:
ok